src/Services/Room/index.js

```diff
@@ -113,13 +113,54 @@ const addVideoToPLaylist = async ({ id, username, link }) => {
   return true;
 }
 
+
+const moveDown = async (roomId, videoId) => {
```

Comment:
```suggestion
const moveDownVideo = async (roomId, videoId) => {
```
Fonksiyonun neyi aşağıya indirdiğini de belirtmeliyiz.